beacon_node/execution_layer/src/engine_api/auth.rs

```diff
@@ -14,7 +14,7 @@ pub const JWT_SECRET_LENGTH: usize = 32;
 #[derive(Debug)]
 pub enum Error {
     JWT(jsonwebtoken::errors::Error),
-    InvalidToken,
+    InvalidToken(reqwest::Error),
```

Comment:
The reason I didn't add the `reqwest` error initially was that the error would always be a `403`
```
Auth(InvalidToken(reqwest::Error { kind: Status(403), url: Url { scheme: "http", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("<DOMAIN>")), port: Some(8551), path: "/", query: None, fragment: None } }))
```
and we aren't really getting any extra information out of the full reqwest error. Not sure if there are cases where we get different errors.

Reply:
ah yeah damn, that doesn't include the body does it, and reqwest's function to get the body as a String is async...